python/src/robyn/modeling/transformations/transformations.py

```diff
@@ -0,0 +1,136 @@
+# pyre-strict
+
+import numpy as np
+import pandas as pd
+from typing import List, Dict, Any, Tuple, Optional
+
+from robyn.data.entities.enums import AdstockType
+
+
+class Transformations:
```

Comment:
All of the methods are static in here. It is not clean object oriented class. Let's use input data as class variables that then is used by methods in this class